person.rb

```diff
@@ -6,10 +6,11 @@ def initialize(age, name = 'unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
   end
 
   attr_accessor :age, name
-  attr_reader :id
+  attr_reader :id, :rental
```

Comment:
- [ ] Hello! 👋 It's awesome that you're including a getter for https://github.com/rentals! 📘 However, it's crucial to keep in mind that since we'll be adding people to rentals, it implies modifications to the rentals object. Without a setter, this could potentially lead to errors. 🚨 Let's make sure to also include a setter for https://github.com/rentals to ensure smooth and error-free updates when adding a people to the rentals. 🛠️ This addition will contribute to the overall stability and functionality of our code! 💻✨

```suggestion
  attr_accessor :date, :book, :person
```